docs/gettingstarted.rst

```diff
@@ -180,7 +180,7 @@ Deploy on CERN infrastructure
 
    .. code-block:: console
 
-      (reana) $ reana-cluster -f reana-cluster.yaml --cvmfs --cephfs --url reana.cern.ch init
+      (reana) $ reana-cluster -f reana-cluster.yaml --cvmfs --cephfs --url reana.cern.ch init --traefik
```

Comment:
We would not need `--traefik` flag for CERN since it comes by default, more info [here](https://clouddocs.web.cern.ch/clouddocs/containers/tutorials/lb.html).